projects/components/src/datagrid/datagrid.component.html

```diff
@@ -88,9 +88,15 @@ <h3 *ngIf="this.header" class="vcd-header">{{ header }}</h3>
             >
             </ng-template>
         </clr-dg-cell>
-        <ng-container ngProjectAs="clr-dg-row-detail" *ngIf="detailTemplate !== undefined">
+        <ng-container ngProjectAs="clr-dg-row-detail" *ngIf="detailComponent !== undefined">
```

Comment:
Since you're touching this, we probably don't need to to check against undefined, just `*ngIf="detailComponent"`